src/zope/testrunner/layer.py

```diff
@@ -13,7 +13,26 @@
 ##############################################################################
 """Layer definitions
 """
+import unittest
 
 
 class UnitTests(object):
     """A layer for gathering all unit tests."""
+
+
+class FakeMultiProcessLayer(object):
+    """A fake layer to start spreading out subprocesses."""
```

Comment:
How about "EmptyLayer"?  There's nothing fake about it ("FakeFoo" is what I name my stubs for use in unit tests; this is a real thing).

The docstring could say "A layer with no tests so subprocesses can start at once" or something like that.